Scripts/Linux/InstallFromSource.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# To test on a Debian-based Linux machine:
+#
+# ./InstallFromSource.sh
+# (outputs Git, GCM, and Scalar versions)
+#
+# Small clone should work:
+# scalar clone https://dev.azure.com/gvfs/ci/_git/ForTests
+#
+# Big clone for authenticated repo should work:
+# scalar clone <url>
+#
+# Note: if you are using a headless Linux machine, then GCM Core will
+# not be able to store your credentials. Instead, create a PAT and
+# use that for password (multiple times). This will be fixed soon.
+
+# Nuke previous data
+rm -rf git-vfs.deb gcm-core scalar log.txt
+
+echo "Installing dotnet SDK 3.1"
+yes | sudo apt-get install dotnet-sdk-3.1 dotnet-runtime-3.1
+
+echo "Installing Git from microsoft/git"
+curl -L https://github.com/microsoft/git/releases/download/v2.28.0.vfs.0.0/git-vfs_2.28.0.vfs.0.0.deb >git-vfs_2.28.0.vfs.0.0.deb
+sudo dpkg -i ./git-vfs_2.28.0.vfs.0.0.deb
+
+echo "Installing Watchman from apt-get"
+sudo apt-get install Watchman 2>1 >>log.txt
```

Comment:
This didn't work for me, as I don't think Watchman is packaged and available in the usual Ubuntu package repositories.  It looks like on the Mac, Homebrew is [packaging](https://formulae.brew.sh/formula/watchman) version 4.9.0 and while there are newer, weekly [binaries](https://facebook.github.io/watchman/docs/install.html#binary-downloads-for-linux-macos-and-windows-beta) for Linux it might make more sense to just use the same stable version (but I don't know if you have a dependency on newer Watchman functionality).  Anyway, assuming 4.9.0 is OK and that we don't need the Python bindings, maybe something like:
```
sudo apt-get install -y autoconf automake build-essential libssl-dev libtool python-dev

git clone https://github.com/facebook/watchman.git -b v4.9.0 --depth 1
cd watchman
git checkout v4.9.0
./autogen.sh 
./configure --without-python
make
sudo make install
```

Plus adding `watchman` to the stuff to be deleted above in the initial `rm -rf` cleanup.

Reply:
I'm going to skip Watchman for now.